pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -169,6 +169,18 @@ cat >/etc/fluentbit/fluentbit.conf <<'EOF'
 	Match journald
 	Rule $LOGKIND ifxaudit ifxaudit false
 
+[FILTER]
+	Name rewrite_tag
+	Match journald
+	Rule $LOGKIND outboundRequests outboundRequests false
+
+[FILTER]
+	Name modify
+	Match  outboundRequests
+	Remove CLIENT_PRINCIPAL_NAME
+	Remove FILE
```

Comment:
Yes, in that table, we don't need/want this field as we just want to see the correlation_id, the message, request time, etc but we are not interested in the file. This table works like some sort of "join table" to correlate inbound requests with outgoing ones so that info is not used 🙂